Module-3-Javascript-Coding-Task/coverage/lcov-report/content-weight.js.html

```diff
@@ -0,0 +1,220 @@
+
+<!doctype html>
+<html lang="en">
+
+<head>
+    <title>Code coverage report for content-weight.js</title>
+    <meta charset="utf-8" />
+    <link rel="stylesheet" href="prettify.css" />
```

Comment:
somehow the coverage folder has been pushed to the repo, please remove it

Reply:
done